.circleci/config.yml

```diff
@@ -0,0 +1,102 @@
+# https://circleci.com/docs/2.0/executor-types/#using-machine
+# Note: Use of machine may require additional fees in a future pricing update.
+
+version: 2
+
+jobs:
+  build:
+    machine: true
+    environment:
+      OPENFISCA_WORKERS: 2
+    steps:
+      - checkout
+      - run:
+          name: Create directory to store artifacts
+          command: mkdir /tmp/artifacts
+      - run:
+          name: Run provisioning
+          command: sudo ./bootstrap.sh origin/master origin/$CIRCLE_BRANCH
+          no_output_timeout: 15m
+      - run:
+          name: Wait for server to start
+          command: sleep 2m
+          no_output_timeout: 3m
+      - run:
+          name: Test node webserver
+          command: wget --retry-connrefused --tries=10 -O /tmp/artifacts/node.out 0.0.0.0:8000 2>/tmp/artifacts/wget.log
+      - run:
+          name: Test node webserver
+          command: cat /tmp/artifacts/node.out | grep --silent 'mes-aides.gouv.fr'
+      - run:
+          name: Test NGINX default server
+          command: wget --retry-connrefused --tries=10 -O /tmp/artifacts/nginx.out 0.0.0.0 2>/tmp/artifacts/wget.log
+      - run:
+          name: Test NGINX default server
```

Comment:
Same here. It took me some time to understand why there are two steps with the same name.